app/js/components/YourSNTBalance.test.jsx

```diff
@@ -5,7 +5,7 @@ import YourSNTBalance from "./YourSNTBalance";
 
 describe('YourSNTBalance', () => {
   it('should render correctly', () => {
-    const component = shallow(<YourSNTBalance value={10}/>);
+    const component = shallow(<YourSNTBalance value={"10"}/>);
```

Comment:
Shouldn't the balance be a number?

Reply:
They are big number and as such represented as string